### ISSUE TITLE
About Transformer Model

### ISSUE THREAD

**Issue by xiang-xiang-zhu** [Shark-NLP/DiffuSeq#28]
I see you use the `self.input_transformers = BertEncoder(config)` to create the Transformer Model, Is the creation of Transformer model realized through BertEncoder? 
Thanks

**Comment by zchoi** [comment]
yes, as far as I'm concerned, it used the BertEncoder with 12 layers transformer encoder layers from huggingface, when `init_pretrained == 'no'`
![image](https://user-images.githubusercontent.com/30498168/220812717-23e37be8-01aa-459d-8af2-196ade1edb01.png)
![image](https://user-images.githubusercontent.com/30498168/220812641-97e8513e-d6d2-4451-9ac2-511879e775c3.png)
